java/impl/src/main/java/net/sf/mmm/code/impl/java/parser/JavaSourceCodeReaderHighlevel.java

```diff
@@ -96,24 +96,6 @@ public BaseType parse(Reader reader, BaseFile javaFile) {
     return javaFile.getType();
   }
 
-  /**
-   * @param reader the {@link Reader} to read the source-code from.
-   * @param javaFile the {@link BaseFile} to read.
-   * @return the full qualified name of the Java file.
-   */
-  public String parseQualifiedName(Reader reader, BaseFile javaFile) {
```

Comment:
Hi @hohwille, 

I gave my point of view about this `parseQualifiedName` method, I'm going to paste it here again just in case you didn't see it, please take it into consideration:

"I really need this feature for the CLI. The use case is that on the CLI I need to load compiled classes which I only know the name of the class, but I don't know the package. The `loadClass(...)` method expects full qualified names.

I tried to search for other solutions on internet, but I didn't find anything which could return easily the full qualified name. In the `mmm-code` library seemed to make sense and it was really easy to implement.

If you don't like it there, please provide me an alternative."

Reply:
Sorry, if I missed this. However, you should be aware and discuss with malte that this "handwritten" Java Source Parser is just an intermediate solution. Malte wants to replace this with an ANTLR generated parser. I started preparing this in a mmm-code-java-parser module but failed to make it work properly.

Reply:
No problem. As Malte is on holidays we will have to postpone this discussion. Unfortunately I need this functionality so I think I won't be able to use the just released `mmm` version :(

Reply:
so if you have a `File` or `Path` refering to a `*.java` file:
Could you create a hackish solution like this:
get the absolute path/file and take the simple name from the Filename excluding the `.java` extension and the package by traversing the path up till you hit `src/*/java`?
So `..../src/main/java/com/example/foo/Bar.java` --> `com.example.foo.Bar`?
Then you would not need mmm-code at all.

Reply:
If that is not doable for some reason, you can still file a feature request issue in mmm-code. However, I am not yet sure how and where to address this as a stable API as the current parser will go away.

Reply:
I have just tried and it is working. I think I can proceed with it.